client/context/query.go

```diff
@@ -31,24 +31,33 @@ func (ctx CLIContext) GetNode() (rpcclient.Client, error) {
 
 // Query performs a query for information about the connected node.
 func (ctx CLIContext) Query(path string, data cmn.HexBytes) (res []byte, err error) {
```

Comment:
Let's remove the named returns here and just return `ctx.query(path, data)`

Reply:
If I update `query` to return height, then should the `Query`, `QueryWithData` also return height?

Reply:
Why not just remove `QueryStoreWithHeight` and have `QueryStore`, `Query`, and `QueryWithData` all return the height.